src/styles/blocks/card.scss

```diff
@@ -0,0 +1,96 @@
+.container {
```

Comment:
it is not part of `card` block so it should be in a separate file (you may rename it to `main` and use `main` tag as container - optionally)